ludwig/utils/dataset_utils.py

```diff
@@ -5,14 +5,15 @@
 
 
 def get_repeatable_train_val_test_split(
-    df_input, stratify_colname, random_seed, frac_train=0.7, frac_val=0.1, frac_test=0.2
+    df_input, stratify_colname="", random_seed=42, frac_train=0.7, frac_val=0.1, frac_test=0.2
```

Comment:
Can we use `from ludwig.utils.defaults import default_random_seed`?

Reply:
good idea, will do